testsuite/smoke-test/src/rosetta.rs

```diff
@@ -1339,8 +1336,10 @@ async fn parse_operations(
                         native_coin(),
                         "Balance should be the native coin"
                     );
-                    let delta =
-                        u64::parse(&amount.value).expect("Should be able to parse amount value");
+                    let delta = amount
+                        .value
+                        .parse::<u64>()
+                        .expect("Should be able to parse amount value");
```

Comment:
FYI changes like this should probably be in a separate commit.

Reply:
These are the changes 😆 The rosetta smoke tests incorrectly relied on the `parse` method from the `PersistentConfig`, so I've cut that dependency and updated the parsing logic 😄